countWorkingDays.ts

```diff
@@ -0,0 +1,41 @@
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+import { holidaysDays } from './holidayTable';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.tz.setDefault('Europe/Warsaw');
+
+const isWeekendOrHoliday = (date: Dayjs): boolean => {
+    const dayOfWeek = date.day();
+    const dateString = date.format('YYYY-MM-DD');
```

Comment:
Proponowałabym Ci format daty wrzucić do osobnego consta - to jest wartość, która jest właściwa konkretnym projektom. 

Albo nawet możesz ten format dać jako default a w wywołaniu `countWorkingDays` umożliwić opcjonalne przekazanie innego formatu.